<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Call</class>
 <widget class="QWidget" name="Call">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Number</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="le_number"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btn_get_contact">
     <property name="text">
      <string>Get Contact</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="le_contact"/>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btn_hangup">
     <property name="text">
      <string>Hangup</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btn_call">
     <property name="text">
      <string>Call</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="btn_mute">
     <property name="text">
      <string>Mute Mic</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="btn_close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="lbl_state">
     <property name="text">
      <string>idle</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="lbl_duration">
     <property name="text">
      <string>00:00</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Duration</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
